dune

```diff
@@ -9,9 +9,9 @@
   (js_of_ocaml
    (compilation_mode separate)
    (flags
-    (:standard --enable effects))
+    (:standard --effects=cps))
```

Comment:
I think we need to keep `--enable effects` here, while `--effects=cps` is not supported by dune.

Reply:
I’m not sure to understand why Dune needs to know about Js_of_ocaml flags?

Reply:
With separate compilation, we don't want to compile libraries (cma files) multiple times when they are used by several executables. But we still have to account for the fact that some flags change the compilation scheme in incompatible way: you cannot link a library compiled with `--enable effects` with some code compiled without this flag, for instance.

Maybe one could add dune options instead of parsing the js_of_ocaml flags. This is [what is done for source maps](https://github.com/ocaml/dune/pull/10777).